CONTRIBUTING.md

```diff
@@ -91,6 +91,8 @@ All icons in Simple Icons have been optimized with the [SVGO tool](https://githu
   * Leave the remaining settings untouched (or reset them with the button at the bottom of the settings).
   * Click the download button.
 
+After optimizing the icon, double-check it against your original version to ensure no visual imperfections have crept in. Also make sure that the dimensions of the path have not been changed so that the icon no longer fits exactly within the canvas, up to 3 decimal points of precision.
```

Comment:
A small change to the end to (maybe) make it a bit clearer.

```suggestion
After optimizing the icon, double check it against your original version to ensure no visual imperfections have crept in. Also make sure that the dimensions of the path have not been changed so that the icon no longer fits exactly within the canvas. We currently check the dimensions up to a precision of 3 decimal points.
```

Reply:
Regarding "double check" or "double-check", the internet suggests to me to use the latter. Then again, you are (I believe) a native speaker 🤔 

Other than that I agree 👍

Reply:
Oop, dunno how that happened. You're correct, it should be hyphenated. I've corrected it above.

Reply:
Alright, done @PeterShaggyNoble 😃